simpleprophet/__init__.py

```diff
@@ -0,0 +1,3 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
We could use some basic documentation, perhaps a README.md in this folder to explain what `simpleprophet` is.

Reply:
It would be helpful for reviewing to have a sense of how this library is invoked. What's the function/functions that you call to make a prediction and persist it?